backend/jsonlogic/filter.go

```diff
@@ -0,0 +1,68 @@
+package jsonlogic
```

Comment:
We should probably add some unit testing here, also around potential SQL injections.

As you note, for now users can't submit this but we should still make sure it's as safe as possible.

Primary reason for unit tests here from my POV is that we don't want to change the results that are produced because that will change the content that is shown. Or at least we should be very aware that we are changing the SQL output here.